vcrt-forwarders.nuspec

```diff
@@ -13,6 +13,9 @@
         <requireLicenseAcceptance>true</requireLicenseAcceptance>
         <description>App-local DLLs that forward to runtime components of Visual C++ libraries which are required to run Visual C++ applications. The runtime components being forwarded to should already be installed on a given machine, this can be done using the Visual C++ Redistributable Packages https://aka.ms/AA4pp63. See https://github.com/Microsoft/vcrt-forwarders for more details. </description>
         <releaseNotes>
+            1.0.8
+            Add support for installing in the unity package Cache
```

Comment:
From the nuget perspective, this isn't really a material change; in a perfect world, this PR would actually not affect the nuget at all. The only reason there's any impact at all is that the .cs script is shared between nuget and UPM, so this comment should be something benign about "revised DLL path discovery logic in Unity initialization script"